README.md

```diff
@@ -204,15 +210,20 @@ The `aws.greengrass.labs.dashboard.Grafana` component supports the following con
                   gD7L5zJLTpzdq23Y/3evLoE=
                   -----END PRIVATE KEY-----
           ```
-4. Create the component by following [the Greengrass documentation](https://docs.aws.amazon.com/greengrass/v2/developerguide/develop-greengrass-components.html)
-   1. Use `zip -r aws-greengrass-labs-dashboard.grafana.zip src` to zip the `src` directory into the deployment artifact, and upload this to S3
-   2. Modify the `aws.greengrass.labs.dashboard.Grafana` recipe at `recipe.yaml`.
-      1. Replace the artifact URI `s3://aws-greengrass-labs-dashboard-grafana.zip` with your S3 path
+6. Create the component by following [the Greengrass documentation](https://docs.aws.amazon.com/greengrass/v2/developerguide/develop-greengrass-components.html)
+   1. Modify the `aws.greengrass.labs.dashboard.Grafana` recipe at `recipe.yaml`.
       2. Replace the two occurrences of `'arn:aws:secretsmanager:<region>:<account>:secret:<name>'` with your created secret ARN.
-   3. (Optional) Specify a mount path. The default used will be `/home/ggc_user/dashboard`.
-      1. When specifying a mount path, note that this mount path will be used to store sensitive data, including secrets and certs used for Grafana auth. You are responsible for securing this directory on your device. Ensure that `ggc_user:ggc_group` has read/write/execute access to this directory with the following command: `namei -m <path>`.
-
-5. Create deployment via the CLI or AWS Console, from [Greengrass documentation](https://docs.aws.amazon.com/greengrass/v2/developerguide/create-deployments.html). The following components should be configured in your deployment:
+      3. (Optional) Specify a mount path. The default used will be `/home/ggc_user/dashboard`.
+         3. When specifying a mount path, note that this mount path will be used to store sensitive data, including secrets and certs used for Grafana auth. You are responsible for securing this directory on your device. Ensure that `ggc_user:ggc_group` has read/write/execute access to this directory with the following command: `namei -m <path>`.
+   2. Use the [GDK CLI](https://docs.aws.amazon.com/greengrass/v2/developerguide/greengrass-development-kit-cli.html) to build the component to prepare for publishing.
+   ```
```

Comment:
add init command to use the component without cloning the repo. 

`gdk component init -r aws-greengrass-labs-dashboard-grafana`